Secure boot / chain of trust 1.15.x Docs (#281)
bottlerocket-os/bottlerocket-project-website: content/en/os/1.15.x/concepts/chain-of-trust/index.markdown

To start, the firmware retrieves the Bottlerocket certificate stored in the system’s NVRAM and uses it to verify that the first-stage bootloader, [shim](https://github.com/rhboot/shim), is correctly signed.

{{% alert title="Note" color="success" %}}
Many operating systems use a version of shim signed by the Microsoft 3rd Party UEFI CA certificate. Bottlerocket does not use this certificate.
Instead, Bottlerocket uses a self-signed certificate intended to allow you to focus your machine’s trust only to your specified OS (in this case, Bottlerocket).
From Microsoft’s article [“Secure the Windows boot process”](https://learn.microsoft.com/en-us/windows/security/operating-system-security/system-security/secure-the-windows-10-boot-process):

*“Since the Microsoft 3rd Party UEFI CA certificate signs the bootloaders for all Linux distributions, trusting the Microsoft 3rd Party UEFI CA signature in the UEFI database increases the attack surface of systems. A customer who intended to only trust and boot a single Linux distribution will trust all distributions - much more than their desired configuration.”*
{{% /alert %}}

> Shim has access to certs stored in NVRAM. These certs are needed to verify the signed binary for GNU Grub, the next step in the boot process.

Reviewer: nit: `GRUB` is the preferred form since it's an acronym

Author: Fixed here and in the diagram.

Grub verifies the signature of its configuration file, `grub.cfg`, then grub calls into shim to verify the kernel.

> The dm-verity root hash for the immutable root filesystem, contains the grub configuration as well. Grub hands off to the now-verified Linux kernel which can mount the root filesystem using with the dm-verity root hash.

Reviewer: ```suggestion
The GRUB configuration also contains the dm-verity root hash for the immutable root filesystem. GRUB hands off to the now-verified Linux kernel which can mount the root filesystem using the dm-verity root hash.
```

Author: works for me!
